src/ralph/lib/table/tests.py

```diff
@@ -13,17 +13,34 @@ def setUpClass(cls):
         super().setUpClass()
         cls.foo_1 = Foo.objects.create(bar='Test1')
 
+    def assertTablesEqual(self, table1, table2):
+        """
+        Compare content of the rows (without checking row type (list vs tuple))
+        """
+        self.assertEqual(len(table1), len(table2))
+        for row1, row2 in zip(table1, table2):
+            self.assertSequenceEqual(row1, row2)
+
     def test_queryset_render(self):
         """Test get_perm_key function."""
         table = Table(Foo.objects.all(), ['id', 'bar'])
         result = [
-            ['ID', 'bar'],
+            [{'value': 'ID'}, {'value': 'bar'}],
```

Comment:
this is verbose name of this field. https://github.com/django/django/blob/master/django/db/models/options.py#L254